global-config/global-config.yaml

```diff
@@ -22,4 +22,5 @@ metadata:
 data:
   mongodb.address: mongodb:27017
   api.host: test.api.ft.com
+  aws.s3.bucket: com.ft.coco-imagepublish.pre-prod
```

Comment:
Maybe rename this to be image-specific? `s3.image.binaries.bucket` or something similar?